### PR TITLE
DPE-2311 Z-Index konsolidieren

### PR DESCRIPTION
**Pull request by Sunny1112358**
Akzeptanzkriterien:

Elemente, die nicht überlagert werden dürfen, sind aufgelistet
Z-Index ist so angepasst, dass keine Fehler entstehen (Hierarchie der Elemente ist korrekt)

### CODE REVIEW

File: tailwind.config.js
Comment (szuelch) [inline tailwind.config.js:1]: Hey @Sunny1112358 , why do you configure z-110 and z-120 with a z-index value of 150 and 160. Please use 110 for z-110 and 120 for z-120.

**Review by szuelch — CHANGES_REQUESTED** [review]
Hi @Sunny1112358, 
please change the config values in tailwind.config.js according to what if written in the comment at that file.

Furthermore please only use yarn as the package manager for this project. So instead of npm install or npm ci use yarn install. I changed that when updating to storybook 7. Please also delete the package.lock.json which was generated when running npm install.

File: tailwind.config.js
Comment (Sunny1112358) [reply tailwind.config.js:1]: yea it was by accident. Thanks for spotting it out

Comment (Sunny1112358) [reply tailwind.config.js:1]: I've been using yarn as well. I'll remove the package lock.

**Review by szuelch — APPROVED** [review]

**Pull request merged**